src/renderer_d3d9.cpp

```diff
@@ -2454,6 +2454,12 @@ namespace bgfx { namespace d3d9
 					bx::read(&reader, texInfo);
 				}
 
+				if (!isShaderVerLess(magic, 10) )
```

Comment:
If shaderc ever gets a rewrite, there should be a mechanism to automatically skip over unknown data in newer shader binaries...